### DISCUSSION TITLE
Our Site Design

### DISCUSSION THREAD

**Discussion by Remian103** [Remian103/lolskin-price-tracker#40] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

디자인에 관련된 의견이나 아이디어 제시 공간

**Comment by jiyolla** [comment]
Carousel에 올릴 때 나타나는 마우스 모양, '잡고 움직일 수 있음'보다 '클릭 가능함'을 암시하는 모양이 더 좋을 거 같음.

전자가 뭔가 '클릭 불가능함'을 내포하는 것 같음.

**Comment by jiyolla** [comment]
로그아웃할 때 확인 있으면 좋을 듯

**Reply by Remian103** [reply]
어떤 확인? 로그아웃하시겠습니까 같은?

**Reply by jiyolla** [reply]
ㅇㅇㅇㅇ

**Reply by Remian103** [reply]
2021.09.13 추가됨

**Comment by jiyolla** [comment]
로그인 버튼 다른 헤더와 같이 한글이면 더 예쁠 듯

**Comment by jiyolla** [comment]
메인 페이지 상단 사이트 이름 폰트 제안:
https://fonts.google.com/specimen/Jost?preview.text=LoL%20Price%20Tracker&preview.text_type=custom&vfonly=true&category=Sans+Serif,Display,Handwriting,Monospace
여기서 Regular 400 어떰.
구글 font치면 나오는 페이지에서 사이트 이름 넣고 폰트 고를 수 있는데, 다른 예쁜 거 제시해도 좋음.

그리고 사이트 이름을 'LoL Price Tracker'로 변경하는 거 어떰. LOL, LoL찾아보니까, LoL을 더 많이 쓰는듯. 혹은 아예 League라고 부르는 듯. 'League Price Tracker'도 괜찮다고 봄. 'PRICE TRACKER'보다는 'Price Tracker'가 좀 더 이쁜 듯.

**Reply by jiyolla** [reply]
지금 폰트도 괜찮은듯? 

현재 폰트 하에서
'LOL PRICE TRACKER'보다는 'LOL Price Tracker'나 'LoL Price Tracker'가 더 예쁘고,
'League Price Tracker' 이 문구가 젤 예쁜 거 같음.

저 링크의 폰트는 대문자 소문자의 차이가 좀 더 강조되는 느낌. 지금 폰트보다 아주 조금 더 이쁘다고 생각함.

**Comment by jiyolla** [comment]
사이트 색상 디자인 제안:
https://mycolor.space/?hex=%230284FE&sub=1
이런 사이트가 있더군!
주제 색깔 하나 정해서, 거기서 제시하는 관련 색으로 현재 UI색상 재구성하면 좋을듯!
원래 쓰던 배경은 그냥 버리거나, 그 배경 자체를 주제색으로 정해서, 색 구성해도 좋을듯.

이게 #01121C를 주제색으로 했을 때, Natural Palette로 고르고, 주제색으로 배경 채우고, 하얀색 비슷한 걸로 제목 채웠을 때 느낌.
![image](https://user-images.githubusercontent.com/71974321/135727916-3f1b0bee-46a0-4607-81fe-c9f28424099d.png)
